lasagne/layers/shape.py

```diff
@@ -21,17 +21,33 @@
 
 class FlattenLayer(Layer):
     """
-    Flatten all but the first dimension.
+    Flatten all dimensions trailing ``dim``
```

Comment:
I think this is ambiguous, to me it sounds like ``dim`` is not included (but it is, right?)